src/main/python/calibration/routes_24_hours.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+DISTANCE="[3000.0,15000.0]"
+SAMPLE_SIZE=1000
+MIN_HOUR=0
+MAX_HOUR=24
+
+function sample () {
+  hour=$1
+  minSeconds=$((hour*3600))
+  maxSeconds=$(((hour+1)*3600-1))
+  departueTime=$(printf '[%d, %d]' $minSeconds $maxSeconds)
+  python3 generate_rides_with_google_maps.py --inputFile=$INPUT_FILE \
+    --travelDistanceIntervalInMeters="${DISTANCE}" --areaBoundBox="${BOUNDING_BOX}" --sampleSize="${SAMPLE_SIZE}" --sampleSeed=12345 --departureTimeIntervalInSeconds="${departueTime}"
+}
+
+function generate() {
```

Comment:
this method is not used

Reply:
it's used in `austin_24hours.sh`, it is included as source